src/components/Menu/index.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
```

Comment:
Ao invés do _import_ separado do useState, você importar numa linha só:
```suggestion
import React, { useState } from "react";
```